test('hello world',()=>{})
test('test 2',()=>{
    throw new Error('fail')
})